tests/blackbox/stratis_cli_cert.py

```diff
@@ -34,6 +34,20 @@
 )
 
 
+def _raise_error_exception(return_code, msg):
+    """
+    Check result of a D-Bus call in a context where it is in error
```

Comment:
Please fix up the header comment so that it is correct for this context: using the CLI rather than directly accessing the D-Bus.

Reply:
Resolved.